wsgi/odyssy/basic/views.py

```diff
@@ -1,5 +1,9 @@
+import os
+
 from django.shortcuts import render
 
+from events.models import Event
 
 def index(request):
-    return render(request, 'basic/index.html')
+    events_list = Event.objects.order_by('-event_date')[:int(os.environ.get('NUM_EVENTS_ON_INDEX_PAGE', 3 ))]
```

Comment:
You have mentioned in #5  that it should be configurable, so are you sure not to use env var?